HTML/exercise-02/script.js

```diff
@@ -0,0 +1,23 @@
+var clock = document.getElementById("clock")
+var toggle  = false
+
+function tick(){
+    clock.innerText = new Date().toLocaleTimeString()
+}
+
+
+
+var toggleBtn = document.getElementById("Toogle")
+var container = document.getElementsByClassName("container")[0]
```

Comment:
container can be declared in the eventlistner for toggle button since it has no use outside this scope